fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -734,6 +1026,11 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 				readValuePrefixTimeout = SERVER_KNOBS->ROCKSDB_READ_VALUE_PREFIX_TIMEOUT;
 				readRangeTimeout = SERVER_KNOBS->ROCKSDB_READ_RANGE_TIMEOUT;
 			}
+			if (SERVER_KNOBS->ROCKSDB_PERFCONTEXT_ENABLE) {
+				// Enable perf context on the same thread with the db thread
+				rocksdb::SetPerfLevel(rocksdb::PerfLevel::kEnableTimeExceptForMutex);
+				rocksdb::get_perf_context()->Reset();
```

Comment:
Can we call perfContextMetrics->reset() method here to be in sync with other places?

Reply:
Yes. I will change.